src/api.ts

```diff
@@ -0,0 +1,15 @@
+const API_URL = 'https://mate.academy/students-api';
+
+export async function getTodos(): Promise<Todo[]> {
+  const response = await fetch(`${API_URL}/todos`);
+  const json = await response.json();
+
+  return json;
```

Comment:
You can return `response.json()`
```suggestion
  return response.json();
```